sanic/config.py

```diff
@@ -1,8 +1,17 @@
-import os
-import types
+from os import environ as os_environ
+
+from typing import Union, \
+                   Any
 
+from .utils import str_to_bool, \
+                   load_module_from_file_location
+
+# TODO: remove in version: 21.3
```

Comment:
Make the comment more explicit as to _what_ needs to be removed. Ideally someone who has never seen this PR would know what to do in January.